SonarQube.Bootstrapper/Program.cs

```diff
@@ -75,20 +82,43 @@ private static int PreProcess(string[] args, IBuildAgentUpdater updater, IBootst
 
             var preprocessorFilePath = settings.PreProcessorFilePath;
             var processRunner = new ProcessRunner();
-            processRunner.Execute(preprocessorFilePath, string.Join(" ", args.Select(a => "\"" + a + "\"")), settings.TempDirectory, logger);
+            processRunner.Execute(preprocessorFilePath, args, settings.TempDirectory, logger);
 
             return processRunner.ExitCode;
         }
 
-        private static int PostProcess(ILogger logger, IBootstrapperSettings settings)
+        private static int PostProcess(string args, IBootstrapperSettings settings, ILogger logger)
```

Comment:
Change of behaviour: the bootstrapper shouldn't know whether the post-processor expects any arguments or not. It should pass them through and let the post-processor handle them. I've updated an existing test to check parameters are passed through.